es-ds-components/components/es-form-msg.vue

```diff
@@ -0,0 +1,69 @@
+<script setup lang="ts">
+const props = defineProps({
+    show: {
+        type: Boolean,
+        default: false,
+    },
+    timeout: {
+        type: Number,
+        default: 20,
+        required: false,
+    },
+    variant: {
+        type: String,
+        default: 'danger',
+        required: false,
+        validator: (val: string) => ['danger', 'success', 'primary'].includes(val),
+    },
+});
+
+const emit = defineEmits(['hidden']);
+
+const hide = () => {
+    emit('hidden');
+}
+
+watch(() => props.show, (newValue, oldValue) => {
```

Comment:
Other instances of `watch` in the code don't include `() =>` so I don't think it's necessary

Reply:
What happens if you remove it? 🙂

Reply:
Ah, I see the warnings now. Would we rather do it with `() =>` or define a ref to the prop to be used for `watch`? That's how it's done in radio button, so we can update that one or this one for consistency.

Reply:
Defining a ref to the prop doesn't work in and of itself, because then the ref is not updated when the prop is updated. Looking at es-radio-button, notice that they also define a watcher on a function that returns the prop, but they use that watcher to update the aforementioned detached ref. I think that's a more roundabout way to do it, and in any case I don't see that it matters in the context of this PR.

Reply:
I think it matters for consistency - I'll make a note to update how it's done in `es-radio-button`

Reply:
I feel like my initial response was unnecessarily sassy 😭 What you didn't realize is how big a percentage of time spent on this code was spent on that line specifically, with me scratching my head over why I couldn't just watch a prop, and what the easiest workaround would be.

Reply:
It was a good learning exercise, and I don't mind a little sass! 💁‍♀️